src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -103,15 +96,7 @@ def run_offline_reintegration_test(self, num_pool, data=False):
                                                        rank,
                                                        t_string)
             self.log.info(output)
-
-            fail_count = 0
-            while fail_count <= 20:
-                pver_reint = self.get_pool_version()
```

Comment:
Same as above. If you remove this `pver_reint`, doesn't it cause problem?

Reply:
Same as above.